sources/packages/api/src/testHelpers/testDBOps.ts

```diff
@@ -0,0 +1,6 @@
+import { Repository, Connection } from "typeorm";
+
+export async function saveInDB<T> (object: T, connection: Connection, entity: any) {
```

Comment:
better will be to name it as 'saveToDB' or simply 'save'

Reply:
Unused code, removed.